docs/book/src/forc/workspaces.md

```diff
@@ -25,6 +26,27 @@ members = ["member1", "path/to/member2"]
 The `members` field accepts entries to be given in relative path with respect to the workspace root.
 Packages that are located within a workspace directory but are *not* contained within the `members` set are ignored.
 
+## The `[patch]` section
+
+The `[patch]` section can be used to override any dependency in the workspace dependency graph. The usage is the same with package level `[patch]` section and details can be seen [here](./manifest_reference.md#the-patch-section).
+
+If a member of a workspace has a patch table in its package manifest, the workspace level patch declarations are merged with package level declarations. If there are conflicting declarations (patching the same source with different target) between workspace level patch table and a member's patch table, workspace level patch declarations.
```

Comment:
I think we may want to follow suit with cargo here and *disallow* package `[patch]` tables if there a workspace `[patch]` table exists.

Patching is specifically relevant to the package graph (represented by the Forc.lock file), and the package graph is always the same within a workspace *regardless* of whether you are only building one member, or many (it's just that when we are building one package, we skip nodes in the graph that are not relevant to the individual package).

Like the `Forc.lock` file, I think we might want to limit the `[patch]` table use to the workspace-level in the case that a workspace exists.